hvpy/api_groups/movies/re_queue_movie.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class reQueueMovieInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the ``reQueueMovie`` API.
```

Comment:
This is not a, fix it in this PR.

For a future issue/PR we should actually turn the reference into a named url and then link it up here.